runtime/Makefile

```diff
@@ -1,17 +1,20 @@
 LLVM_SUFFIX?=-11
 export CLANG ?= clang$(LLVM_SUFFIX)
 LLVM_AS ?= llvm-as$(LLVM_SUFFIX)
+LLC ?= llc$(LLVM_SUFFIX)
 export LLVM_LINK ?= llvm-link$(LLVM_SUFFIX)
 VALGRIND ?= valgrind
 OBJS=alloc.o list.o main.o mapping.o panic.o print.o string.o int.o third-party/dtoa/emyg_dtoa.o
+RT_BCS=alloc.bc list.bc main.bc mapping.bc panic.bc print.bc string.bc int.bc third-party/dtoa/emyg_dtoa.bc
```

Comment:
ideally we shouldn't need to all the files again.

Reply:
I removed as much as possible, and removing rest of the files occur errors when running make.